pytom/gui/guiFunctions.py

```diff
@@ -472,8 +475,7 @@ def avail_gpu(cutoff_busy=.25, cutoff_space = 0.5):
     @param cutoff_space: fraction of memeory of GPU available
     @return: list of gpu indices of available gpus
     '''
-    lines = [line.split() for line in os.popen('nvidia-smi').readlines()]
-
+    lines = [line.split() for line in subprocess.run(['nvidia-smi'], capture_output=True, text=True).stdout.splitlines()]
```

Comment:
Looking at this whole function it would be better to switch to something like this : https://docs.cupy.dev/en/stable/reference/generated/cupy.cuda.runtime.getDeviceCount.html

This function even only allows 'GeForce' cards looking at line 482 :|

Reply:
I agree, split it off to issue #170